utils/zerovec/src/hashmap/mod.rs

```diff
@@ -0,0 +1,7 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+#[allow(clippy::module_inception)] // module is purely internal
+mod hashmap;
+pub use hashmap::ZeroHashMapStatic;
```

Comment:
we should have the code that is in hashmap.rs live in mod.rs

(we can split it again later if we really need to)